app/src/routes/login/login.vue

```diff
@@ -78,7 +79,11 @@ export default defineComponent({
 			providerOptions.value = providers.value
 				.filter((provider) => !AUTH_SSO_DRIVERS.includes(provider.driver))
 				.map((provider) => ({ text: formatTitle(provider.name), value: provider.name }));
-			providerOptions.value.unshift({ text: t('default_provider'), value: DEFAULT_AUTH_PROVIDER });
+			if (!disableDefault.value) {
+				providerOptions.value.unshift({ text: t('default_provider'), value: DEFAULT_AUTH_PROVIDER });
+			} else {
+				providerSelect.value = providerOptions.value[0]?.value;
+			}
```

Comment:
I know it's technically semi-unrelated to this particular feature, but lets move this whole watch block callback function into the fetchProviders try/catch. The providers won't change during regular usage of the app, and are only fetched one time when the login form is rendered. No need to have a set of dynamic refs and watchers to handle that 👍🏻

Reply:
```ts
providerOptions.value = providers.value
	.filter((provider) => !AUTH_SSO_DRIVERS.includes(provider.driver))
	.map((provider) => ({ text: formatTitle(provider.name), value: provider.name }));
```

Should I also move this then? Because otherwise I run into the issue where the selector defaults to default even though it is not in `providerOptions`

Reply:
Sorry, whole watch block includes this :sweat_smile: 
Nevermind

Reply:
Should be resolved in a20e830